Clicking Edit Data row number causes SlickGrid exception

<!-- Please search existing issues to avoid creating duplicates. -->
<!-- Also please test using the latest insiders build to make sure your issue has not already been fixed. -->

<!-- Use Help > Report Issue to prefill these. -->
- SQL Operations Studio Version:

Steps to Reproduce:

1. Edit Data on a table
2. Click the row number

Exception
```
D:\xplat\sqlopsstudio\node_modules\zone.js\dist\zone-node.js:2280 [uncaught exception]: Error: Selection model is not set
console.(anonymous function) @ D:\xplat\sqlopsstudio\node_modules\zone.js\dist\zone-node.js:2280
D:\xplat\sqlopsstudio\node_modules\zone.js\dist\zone-node.js:2280 Error: Selection model is not set
    at SlickGrid.setSelectedRows (D:\xplat\sqlopsstudio\node_modules\slickgrid\slick.grid.js:3835:15)
    at RowNumberColumn.handleClick (file:///D:/xplat/sqlopsstudio/out/sql/base/browser/ui/table/plugins/rowNumberColumn.plugin.js:27:27)
    at SlickGrid.<anonymous> (file:///D:/xplat/sqlopsstudio/out/sql/base/browser/ui/table/plugins/rowNumberColumn.plugin.js:18:81)
    at Event.notify (D:\xplat\sqlopsstudio\node_modules\slickgrid\slick.core.js:145:35)
    at trigger (D:\xplat\sqlopsstudio\node_modules\slickgrid\slick.grid.js:1235:18)
    at HTMLDivElement.handleClick (D:\xplat\sqlopsstudio\node_modules\slickgrid\slick.grid.js:2744:7)
    at HTMLDivElement.dispatch (D:\xplat\sqlopsstudio\node_modules\jquery\dist\jquery.js:5110:27)
    at HTMLDivElement.$event.dispatch (file:///D:/xplat/sqlopsstudio/node_modules/slickgrid/lib/jquery.event.drag-2.3.0.js:382:30)
    at HTMLDivElement.elemData.handle 
```

--- a/src/sql/base/browser/ui/table/plugins/rowNumberColumn.plugin.ts
+++ b/src/sql/base/browser/ui/table/plugins/rowNumberColumn.plugin.ts
@@ -36,14 +36,18 @@ export class RowNumberColumn<T> implements Slick.Plugin<T> {
 	private handleClick(e: MouseEvent, args: Slick.OnClickEventArgs<T>): void {
 		if (this.grid.getColumns()[args.cell].id === 'rowNumber') {
 			this.grid.setActiveCell(args.row, 1);
-			this.grid.setSelectedRows([args.row]);
+			if (this.grid.getSelectionModel()) {
+				this.grid.setSelectedRows([args.row]);
+			}
 		}
 	}
 
 	private handleHeaderClick(e: MouseEvent, args: Slick.OnHeaderClickEventArgs<T>): void {
 		if (args.column.id === 'rowNumber') {
 			this.grid.setActiveCell(0, 1);
-			this.grid.setSelectedRows(range(this.grid.getDataLength()));
+			if (this.grid.getSelectionModel()) {
+				this.grid.setSelectedRows(range(this.grid.getDataLength()));
+			}
 		}
 	}
 

